src/modules/tokens/views/Accounts/AccountDetails.tsx

```diff
@@ -246,9 +323,21 @@ export const AccountDetails: () => JSX.Element = () => {
             >
               {t('accounts')}
             </Link>
-            <Link underline="none" color="text.primary">
+            <Link
+              underline="hover"
+              color="inherit"
+              sx={{ cursor: 'pointer' }}
+              onClick={() =>
+                history.push(
+                  `/namespace/${selectedNamespace}/tokens/accounts/${key}`
+                )
+              }
+            >
               {key}
```

Comment:
do we want to use short keys here?

Reply:
do we have a way to shorten the key without triggering a popover? the link here should just be to the previous page.

Reply:
Probably can make a util function to shorten it